crates/forge/src/run_tests/resolve_config.rs

```diff
@@ -78,27 +77,6 @@ async fn resolve_fork_config(
     Ok(Some(ResolvedForkConfig { url, block_number }))
 }
 
-fn parse_block_id(fork_target: &ForkTarget) -> Result<BlockId> {
-    let block_id = match fork_target.block_id_type.as_str() {
-        "number" => BlockId::BlockNumber(fork_target.block_id_value.parse()?),
-        "hash" => {
-            let block_hash = fork_target.block_id_value.parse::<BigInt>()?;
-
-            BlockId::BlockHash(block_hash.into())
-        }
```

Comment:
It seems like previously it was possible to pass felt as either decimal or hex. We should verify if that was true and if so if this is still possible

Reply:
It is possible, I've added tests